### PR TITLE
Modified openapi/platform/platform.yaml

### PR DESCRIPTION
**Pull request by agopalan87**
@vendasta/external-apis
https://vendasta.jira.com/wiki/spaces/API/pages/edit-v2/1808596993

**TODO**:
[ ] Review the [pre release checklist](https://vendasta.jira.com/wiki/spaces/API/pages/1651769533/Pre+Release+Checks)

### CODE REVIEW

**Review by richard-rance — CHANGES_REQUESTED** [review]
There are a lot of unrelated (and unwanted) changes to the proposals related sections. Please pair this down to just he `Content-Type` changes.

File: openapi/platform/platform.yaml [hunk 1]
```diff
@@ -4441,7 +4441,9 @@ paths:
       description: |-
         [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
 
-        Provides a list of products that can be activated by a partner for their customers. You must filter the list to a single partner using the `filter[partner.id]=PID` query param.
+        Provides a list of products that can be activated by a partner for their customers. 
+        You must filter the list to a single partner using the `filter[partner.id]=PID` query param. 
+        Sorted by the whiteLabelName field in ascending order.
```
Comment (richard-rance) [inline openapi/platform/platform.yaml:4446]: Are you sure that it is the white label name and not the vendor's name? That is great if that is correct, I'm just surprised.

**Review by richard-rance — APPROVED** [review]
Just a couple of minor things to adjust

File: openapi/platform/platform.yaml (hunk 1 above)
Comment (agopalan87) [reply openapi/platform/platform.yaml:4446]: Yes. The internal API currently provides only the Partner given white label name for a product (if white labelling is done). We are retrieving vendor name from a different API.

File: openapi/platform/platform.yaml [hunk 2]
```diff
@@ -7436,7 +7394,26 @@ components:
...
+            bannerImageUrl: 'http://example.com'
+          relationships:
+            allowedCountries:
+              data:
+                - id: US
+                  type: countries
+            requiredProductIds:
+              data:
+                - id: 'MP-456:EDITION-456'
+                  type: partnerActivatableProducts
```
[the remark this answers is not in the document]
Comment (richard-rance) [reply openapi/platform/platform.yaml]: That is strange. I had never noticed that it converts to yaml for storage. You can merge this PR.

**Pull request merged**